src/commands/convert/hdf2xccdf.ts

```diff
@@ -12,8 +12,8 @@ export default class HDF2XCCDF extends Command {
 
   static flags = {
     help: Flags.help({char: 'h'}),
-    input: Flags.string({char: 'i', required: true, description: 'Input HDF file'}),
-    output: Flags.string({char: 'o', required: true, description: 'Output HDF JSON File'}),
+    input: Flags.string({char: 'i', required: true, description: 'Input HDF JSON file'}),
+    output: Flags.string({char: 'o', required: true, description: 'Output XCCDF XML File'}),
```

Comment:
Super nitpick - the file is lowercase and the second one is uppercase. I am assuming lowercase is the consistent choice.

Reply:
```suggestion
    output: Flags.string({char: 'o', required: true, description: 'Output XCCDF XML file'}),
```